Update status checker tool to support NO_IAF

siyer-123 opened the pull request.

lsalazar1 commented:
Hi Sid, can you please help me understand what is wrong with the status tool?

```console
[root@api.pours.cp.fyre.ibm.com cp4waiops-samples]# oc waiops status
No resources found
No resources found
error: flag needs an argument: 'n' in -n
See 'oc get --help' for usage.
error: the server doesn't have a resource type "jsonpath={"
Using project "katamari" on server "https://api.cp4waiops-rtp-26014519.cp.fyre.ibm.com:6443".

Cloud Pak for Watson AIOps AI Manager v installation status:
______________________________________________________________
Installation instances:

No resources found in katamari namespace.
error: error executing jsonpath "{.items[].status.phase}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[].status.phase}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}




______________________________________________________________
ZenService instances:

error: the server doesn't have a resource type "zenservice"
error: the server doesn't have a resource type "zenservice"


______________________________________________________________
AutomationUIConfig, AutomationBase, Cartridge, CartridgeRequirements, and EventProcessor instances:

error: the server doesn't have a resource type "automationuiconfig"
error: the server doesn't have a resource type "automationuiconfig"
error: the server doesn't have a resource type "automationuiconfig"


error: the server doesn't have a resource type "automationbase"
error: the server doesn't have a resource type "automationbase"
error: the server doesn't have a resource type "automationbase"


error: the server doesn't have a resource type "cartridge"
error: the server doesn't have a resource type "cartridge"


error: the server doesn't have a resource type "cartridgerequirements"
error: the server doesn't have a resource type "cartridgerequirements"


error: the server doesn't have a resource type "eventprocessor"
error: the server doesn't have a resource type "eventprocessor"


error: the server doesn't have a resource type "eventprocessor"
error: the server doesn't have a resource type "eventprocessor"


______________________________________________________________
IRCore and AIOpsAnalyticsOrchestrator instances:

error: the server doesn't have a resource type "ircore"
error: the server doesn't have a resource type "ircore"


error: the server doesn't have a resource type "AIOpsAnalyticsOrchestrator"
error: the server doesn't have a resource type "AIOpsAnalyticsOrchestrator"


______________________________________________________________
LifecycleService instances:

error: the server doesn't have a resource type "lifecycleservice"
error: the server doesn't have a resource type "lifecycleservice"


______________________________________________________________
BaseUI instances:

error: the server doesn't have a resource type "BaseUI"
error: the server doesn't have a resource type "BaseUI"


______________________________________________________________
AIManager, ASM, AIOpsEdge, and AIOpsUI instances:

error: the server doesn't have a resource type "AIManager"
error: the server doesn't have a resource type "AIManager"


error: the server doesn't have a resource type "asm"
error: the server doesn't have a resource type "asm"


error: the server doesn't have a resource type "aiopsedge"
error: the server doesn't have a resource type "aiopsedge"


error: the server doesn't have a resource type "aiopsui"
error: the server doesn't have a resource type "aiopsui"


______________________________________________________________

Hint: for a more detailed printout of component statuses, run `oc waiops status-all`.

[root@api.pours.cp.fyre.ibm.com cp4waiops-samples]# 

```

For some context, I do have an 0.1-Dev build on this system. Seems like there is an issue fetching two resources in the beginning.

lsalazar1 commented:
Hi Sid,

I see in your second bullet point that the status tool will not throw errors if NO_IAF is on. I'm using a 0.1 dev build from yesterday but I'm noticing that errors still pop up on my end.

```console
[root@<redacted> cp4waiops-samples]# oc get $CSV_NAME -n $INSTALLATION_NAMESPACE -o jsonpath='{.spec.version}' | awk '{ print substr( $0, 0, 3 ) }'
0.1
[root@<redeacted> cp4waiops-samples]# oc waiops status
Already on project "katamari" on server "https://api.<redacted>:6443".

Cloud Pak for Watson AIOps AI Manager v0.1 installation status:
______________________________________________________________
Installation instances:

NAME                  PHASE      LICENSE    STORAGECLASS   STORAGECLASSLARGEBLOCK   AGE
ibm-cp-watson-aiops   Updating   Accepted   rook-cephfs    rook-ceph-rbd            24m

______________________________________________________________
ZenService instances:

KIND         NAME                 NAMESPACE   VERSION   STATUS       PROGRESS   MESSAGE
ZenService   iaf-zen-cpdservice   katamari    <none>    InProgress   98%        Finished Role 0030-gateway

______________________________________________________________
AutomationUIConfig, AutomationBase, Cartridge, CartridgeRequirements, and EventProcessor instances:

error: the server doesn't have a resource type "automationuiconfig"
error: the server doesn't have a resource type "automationuiconfig"
error: the server doesn't have a resource type "automationuiconfig"


error: the server doesn't have a resource type "automationbase"
error: the server doesn't have a resource type "automationbase"
error: the server doesn't have a resource type "automationbase"


error: the server doesn't have a resource type "cartridge"
error: the server doesn't have a resource type "cartridge"


error: the server doesn't have a resource type "cartridgerequirements"
error: the server doesn't have a resource type "cartridgerequirements"


error: the server doesn't have a resource type "eventprocessor"
error: the server doesn't have a resource type "eventprocessor"


error: the server doesn't have a resource type "eventprocessor"
error: the server doesn't have a resource type "eventprocessor"


______________________________________________________________
IRCore and AIOpsAnalyticsOrchestrator instances:

KIND                  NAMESPACE   NAME    VERSION   STATUS   MESSAGE
IssueResolutionCore   katamari    aiops   0.1.0     Ready    All Services Ready

KIND                         NAMESPACE   NAME    VERSION   STATUS     MESSAGE
AIOpsAnalyticsOrchestrator   katamari    aiops             Updating   Update in progress

______________________________________________________________
LifecycleService instances:

KIND               NAMESPACE   NAME    VERSION   STATUS   MESSAGE
LifecycleService   katamari    aiops   0.1.0     <none>   All Services Ready

______________________________________________________________
BaseUI instances:

KIND     NAMESPACE   NAME              VERSION   STATUS   MESSAGE
BaseUI   katamari    baseui-instance   0.1.0     False    Not ready

______________________________________________________________
AIManager, ASM, AIOpsEdge, and AIOpsUI instances:

KIND        NAMESPACE   NAME        VERSION   STATUS        MESSAGE
AIManager   katamari    aimanager   4.1.0     Reconciling   Waiting for zen to be in a ready state

KIND   NAMESPACE   NAME             VERSION   STATUS
ASM    katamari    aiops-topology   2.14.0    OK

KIND        NAMESPACE   NAME        STATUS       MESSAGE
AIOpsEdge   katamari    aiopsedge   Configured   all critical components are reporting healthy

KIND      NAMESPACE   NAME               VERSION   STATUS   MESSAGE
AIOpsUI   katamari    aiopsui-instance   0.1.0     False    Not ready

______________________________________________________________

Hint: for a more detailed printout of component statuses, run `oc waiops status-all`.

```

lsalazar1 reviewed (approved):
LGTM

```console
[root@api.pours.cp.fyre.ibm.com kubectl-plugin]# oc waiops status
Already on project "katamari" on server "https://api.cp4waiops-rtp-26014519.cp.fyre.ibm.com:6443".

Cloud Pak for Watson AIOps AI Manager v0.1 installation status:
______________________________________________________________
Installation instances:

NAME                  PHASE      LICENSE    STORAGECLASS   STORAGECLASSLARGEBLOCK   AGE
ibm-cp-watson-aiops   Updating   Accepted   rook-cephfs    rook-ceph-rbd            34m

______________________________________________________________
ZenService instances:

KIND         NAME                 NAMESPACE   VERSION   STATUS      PROGRESS   MESSAGE
ZenService   iaf-zen-cpdservice   katamari    4.8.4     Completed   100%       The Current Operation Is Completed

______________________________________________________________
IRCore and AIOpsAnalyticsOrchestrator instances:

KIND                  NAMESPACE   NAME    VERSION   STATUS   MESSAGE
IssueResolutionCore   katamari    aiops   0.1.0     Ready    All Services Ready

KIND                         NAMESPACE   NAME    VERSION   STATUS     MESSAGE
AIOpsAnalyticsOrchestrator   katamari    aiops             Updating   Update in progress

______________________________________________________________
LifecycleService instances:

KIND               NAMESPACE   NAME    VERSION   STATUS   MESSAGE
LifecycleService   katamari    aiops   0.1.0     <none>   All Services Ready

______________________________________________________________
BaseUI instances:

KIND     NAMESPACE   NAME              VERSION   STATUS   MESSAGE
BaseUI   katamari    baseui-instance   0.1.0     True     Ready

______________________________________________________________
AIManager, ASM, AIOpsEdge, and AIOpsUI instances:

KIND        NAMESPACE   NAME        VERSION   STATUS      MESSAGE
AIManager   katamari    aimanager   4.1.0     Completed   AI Manager is ready

KIND   NAMESPACE   NAME             VERSION   STATUS
ASM    katamari    aiops-topology   2.14.0    OK

KIND        NAMESPACE   NAME        STATUS       MESSAGE
AIOpsEdge   katamari    aiopsedge   Configured   all critical components are reporting healthy

KIND      NAMESPACE   NAME               VERSION   STATUS   MESSAGE
AIOpsUI   katamari    aiopsui-instance   0.1.0     True     Ready

______________________________________________________________

Hint: for a more detailed printout of component statuses, run `oc waiops status-all`.
```

siyer-123 commented:
Going to add kafka and elastic back in since we are using the Bedrock one instead of the IAF one

siyer-123 commented:
Updated `status` and `status-all` to include a Kafka and Elasticsearch section:
<img width="816" alt="Screenshot 2023-06-07 at 11 42 31 AM" src="https://github.com/IBM/cp4waiops-samples/assets/91081503/a4085f60-365e-4059-8e90-a661143915a7">

Updated `status-upgrade` to reflect versions in release-41 branch (note for AIManager failing, AIManager's version should be 4.1; in 3.7 its version was 3.7, so this is correct. For ASM, its version in 3.7 was 2.15, so we will have to update this if needed later):
<img width="772" alt="Screenshot 2023-06-07 at 11 44 27 AM" src="https://github.com/IBM/cp4waiops-samples/assets/91081503/b32abd24-090c-4591-b686-de5b756c8890">

lsalazar1 reviewed (approved):
lgtm

siyer-123 commented:
Updated to address ibm-elastic-operator operandrequest change noted by Bryan. Results look good now:

```
[root@api.redraws.cp.fyre.ibm.com ~]# oc waiops status-all
Already on project "cp4waiops" on server "https://c107-e.us-south.containers.cloud.ibm.com:30113".

Cloud Pak for Watson AIOps AI Manager v3.7 installation status:
______________________________________________________________
Installation instances:

NAME                  PHASE     LICENSE    STORAGECLASS         STORAGECLASSLARGEBLOCK   AGE
ibm-cp-watson-aiops   Running   Accepted   ibmc-file-gold-gid   ibmc-block-gold          126m

______________________________________________________________
ZenService instances:

KIND         NAME                 NAMESPACE   VERSION   STATUS      PROGRESS   MESSAGE
ZenService   iaf-zen-cpdservice   cp4waiops   4.8.5     Completed   100%       The Current Operation Is Completed

______________________________________________________________
AutomationUIConfig, AutomationBase, Cartridge, CartridgeRequirements, and EventProcessor instances:

KIND                 NAMESPACE   NAME         VERSION   STATUS   MESSAGE
AutomationUIConfig   cp4waiops   iaf-system   1.3.12    True     AutomationUIConfig successfully registered

KIND             NAMESPACE   NAME                    VERSION   STATUS   MESSAGE
AutomationBase   cp4waiops   automationbase-sample   2.0.12    True     AutomationBase instance successfully created

KIND        NAMESPACE   NAME                  VERSION   STATUS   MESSAGE
Cartridge   cp4waiops   cp4waiops-cartridge   1.3.12    True     Cartridge successfully registered

KIND                    NAMESPACE   NAME                  VERSION   STATUS   MESSAGE
CartridgeRequirements   cp4waiops   cp4waiops-cartridge   1.3.12    True     CartridgeRequirements successfully registered

KIND             NAMESPACE   NAME                 VERSION   STATUS   MESSAGE
EventProcessor   cp4waiops   aiops-ir-lifecycle   3.0.0     True     Event Processor is ready

KIND             NAMESPACE   NAME                       VERSION   STATUS   MESSAGE
EventProcessor   cp4waiops   cp4waiops-eventprocessor   4.0.13    True     Event Processor is ready

______________________________________________________________
IRCore and AIOpsAnalyticsOrchestrator instances:

KIND                  NAMESPACE   NAME    VERSION   STATUS   MESSAGE
IssueResolutionCore   cp4waiops   aiops   3.7.2     Ready    All Services Ready

KIND                         NAMESPACE   NAME    VERSION   STATUS   MESSAGE
AIOpsAnalyticsOrchestrator   cp4waiops   aiops   3.7.2     Ready    All Services Ready

______________________________________________________________
LifecycleService instances:

KIND               NAMESPACE   NAME    VERSION   STATUS   MESSAGE
LifecycleService   cp4waiops   aiops   3.7.2     <none>   All Services Ready

______________________________________________________________
BaseUI instances:

KIND     NAMESPACE   NAME              VERSION   STATUS   MESSAGE
BaseUI   cp4waiops   baseui-instance   3.7.2     True     Ready

______________________________________________________________
AIManager, ASM, AIOpsEdge, and AIOpsUI instances:

KIND        NAMESPACE   NAME        VERSION   STATUS      MESSAGE
AIManager   cp4waiops   aimanager   2.8.2     Completed   AI Manager is ready

KIND   NAMESPACE   NAME             VERSION   STATUS
ASM    cp4waiops   aiops-topology   2.13.2    OK

KIND        NAMESPACE   NAME        STATUS       MESSAGE
AIOpsEdge   cp4waiops   aiopsedge   Configured   all critical components are reporting healthy

KIND      NAMESPACE   NAME               VERSION   STATUS   MESSAGE
AIOpsUI   cp4waiops   aiopsui-instance   3.7.2     True     Ready

______________________________________________________________
Kong instances:

KIND   NAMESPACE   NAME      STATUS   MESSAGE
Kong   cp4waiops   gateway   True     <none>

______________________________________________________________
Vault (VaultDeploy and VaultAccess) instances:

KIND          NAMESPACE   NAME               VERSION   STATUS   MESSAGE
VaultDeploy   cp4waiops   ibm-vault-deploy   3.7.2     True     VaultDeploy completed successfully

KIND          NAMESPACE   NAME               VERSION   STATUS   MESSAGE
VaultAccess   cp4waiops   ibm-vault-access   3.7.2     True     VaultAccess completed successfully

______________________________________________________________
Postgres instances:

KIND      NAMESPACE   NAME                               STATUS
Cluster   cp4waiops   ibm-cp-watson-aiops-edb-postgres   Cluster in healthy state

______________________________________________________________
Secure Tunnel instances:

KIND     NAMESPACE   NAME         STATUS
Tunnel   cp4waiops   sre-tunnel   True

______________________________________________________________
CSVs from cp4waiops namespace:

NAME                        DISPLAY                       VERSION   REPLACES   PHASE
aimanager-operator.v3.7.2   IBM Watson AIOps AI Manager   3.7.2                Succeeded

NAME                        DISPLAY                 VERSION   REPLACES   PHASE
aiopsedge-operator.v3.7.2   IBM Watson AIOps Edge   3.7.2                Succeeded

NAME                  DISPLAY                             VERSION   REPLACES   PHASE
asm-operator.v3.7.2   IBM Netcool Agile Service Manager   3.7.2                Succeeded

NAME                     DISPLAY                                            VERSION   REPLACES   PHASE
ibm-aiops-ir-ai.v3.7.2   IBM Watson AIOps Issue Resolution AI & Analytics   3.7.2                Succeeded

NAME                       DISPLAY                                  VERSION   REPLACES   PHASE
ibm-aiops-ir-core.v3.7.2   IBM Watson AIOps Issue Resolution Core   3.7.2                Succeeded

NAME                            DISPLAY                                    VERSION   REPLACES   PHASE
ibm-aiops-ir-lifecycle.v3.7.2   IBM Cloud Pak for Watson AIOps Lifecycle   3.7.2                Succeeded

NAME                            DISPLAY                                     VERSION   REPLACES   PHASE
ibm-aiops-orchestrator.v3.7.2   IBM Cloud Pak for Watson AIOps AI Manager   3.7.2                Succeeded

NAME                          DISPLAY                          VERSION   REPLACES                      PHASE
ibm-automation-core.v1.3.14   IBM Automation Foundation Core   1.3.14    ibm-automation-core.v1.3.13   Succeeded

NAME                                     DISPLAY                                      VERSION   REPLACES                                 PHASE
ibm-automation-eventprocessing.v1.3.14   IBM Automation Foundation Event Processing   1.3.14    ibm-automation-eventprocessing.v1.3.13   Succeeded

NAME                     DISPLAY                     VERSION   REPLACES                 PHASE
ibm-automation.v1.3.14   IBM Automation Foundation   1.3.14    ibm-automation.v1.3.13   Succeeded

NAME                             DISPLAY       VERSION   REPLACES                         PHASE
ibm-automation-elastic.v1.3.13   IBM Elastic   1.3.13    ibm-automation-elastic.v1.3.12   Succeeded

NAME                           DISPLAY                           VERSION   REPLACES                       PHASE
ibm-automation-flink.v1.3.13   IBM Automation Foundation Flink   1.3.13    ibm-automation-flink.v1.3.12   Succeeded

NAME                               DISPLAY                  VERSION   REPLACES                           PHASE
ibm-cloud-databases-redis.v1.4.3   IBM Operator for Redis   1.4.3     ibm-cloud-databases-redis.v1.4.2   Succeeded

NAME                                  DISPLAY                               VERSION   REPLACES   PHASE
ibm-common-service-operator.v3.23.4   IBM Cloud Pak foundational services   3.23.4               Succeeded

NAME                         DISPLAY                                VERSION   REPLACES   PHASE
ibm-management-kong.v3.7.2   IBM Internal - IBM Watson AIOps Kong   3.7.2                Succeeded

NAME                                DISPLAY             VERSION   REPLACES   PHASE
ibm-secure-tunnel-operator.v3.7.2   IBM Secure Tunnel   3.7.2                Succeeded

NAME                        DISPLAY              VERSION   REPLACES   PHASE
ibm-vault-operator.v3.7.2   IBM Vault Operator   3.7.2                Succeeded

NAME                                  DISPLAY               VERSION   REPLACES   PHASE
ibm-watson-aiops-ui-operator.v3.7.2   IBM Watson AIOps UI   3.7.2                Succeeded

______________________________________________________________
CSVs from ibm-common-services namespace:

NAME                              DISPLAY                       VERSION   REPLACES                          PHASE
cloud-native-postgresql.v1.18.4   EDB Postgres for Kubernetes   1.18.4    cloud-native-postgresql.v1.18.3   Succeeded

NAME                                DISPLAY            VERSION   REPLACES   PHASE
ibm-cert-manager-operator.v3.25.4   IBM Cert Manager   3.25.4               Succeeded

NAME                                  DISPLAY                               VERSION   REPLACES   PHASE
ibm-common-service-operator.v3.23.4   IBM Cloud Pak foundational services   3.23.4               Succeeded

NAME                            DISPLAY         VERSION   REPLACES   PHASE
ibm-commonui-operator.v1.21.4   Ibm Common UI   1.21.4               Succeeded

NAME                              DISPLAY          VERSION   REPLACES   PHASE
ibm-crossplane-operator.v1.12.4   IBM Crossplane   1.12.4               Succeeded

NAME                                                  DISPLAY                              VERSION   REPLACES   PHASE
ibm-crossplane-provider-kubernetes-operator.v1.12.4   IBM Crossplane Provider Kubernetes   1.12.4               Succeeded

NAME                         DISPLAY               VERSION   REPLACES   PHASE
ibm-events-operator.v4.6.1   IBM Events Operator   4.6.1                Succeeded

NAME                       DISPLAY   VERSION   REPLACES   PHASE
ibm-iam-operator.v3.23.4   IBM IAM   3.23.4               Succeeded

NAME                                 DISPLAY                      VERSION   REPLACES   PHASE
ibm-ingress-nginx-operator.v1.20.4   IBM Ingress Nginx Operator   1.20.4               Succeeded

NAME                             DISPLAY         VERSION   REPLACES   PHASE
ibm-licensing-operator.v1.20.4   IBM Licensing   1.20.4               Succeeded

NAME                                      DISPLAY              VERSION   REPLACES   PHASE
ibm-management-ingress-operator.v1.20.4   Management Ingress   1.20.4               Succeeded

NAME                           DISPLAY                VERSION   REPLACES   PHASE
ibm-mongodb-operator.v1.18.4   IBM MongoDB Operator   1.18.4               Succeeded

NAME                                   DISPLAY                       VERSION   REPLACES   PHASE
ibm-namespace-scope-operator.v1.17.4   IBM NamespaceScope Operator   1.17.4               Succeeded

NAME                                DISPLAY            VERSION   REPLACES   PHASE
ibm-platform-api-operator.v3.25.4   IBM Platform API   3.25.4               Succeeded

NAME                      DISPLAY           VERSION   REPLACES   PHASE
ibm-zen-operator.v1.8.5   IBM Zen Service   1.8.5                Succeeded

NAME                                           DISPLAY                                VERSION   REPLACES   PHASE
operand-deployment-lifecycle-manager.v1.21.4   Operand Deployment Lifecycle Manager   1.21.4               Succeeded

______________________________________________________________
Subscriptions from cp4waiops namespace:

NAME                 PACKAGE              SOURCE                 CHANNEL
aimanager-operator   aimanager-operator   ibm-operator-catalog   v3.7

NAME                 PACKAGE              SOURCE                 CHANNEL
aiopsedge-operator   aiopsedge-operator   ibm-operator-catalog   v3.7

NAME           PACKAGE        SOURCE                 CHANNEL
asm-operator   asm-operator   ibm-operator-catalog   v3.7

NAME                     PACKAGE                  SOURCE                 CHANNEL
ibm-aiops-orchestrator   ibm-aiops-orchestrator   ibm-operator-catalog   v3.7

NAME                                                             PACKAGE          SOURCE                 CHANNEL
ibm-automation-v1.3-ibm-operator-catalog-openshift-marketplace   ibm-automation   ibm-operator-catalog   v1.3

NAME                                                                  PACKAGE               SOURCE                 CHANNEL
ibm-automation-core-v1.3-ibm-operator-catalog-openshift-marketplace   ibm-automation-core   ibm-operator-catalog   v1.3

NAME                                                                             PACKAGE                          SOURCE                 CHANNEL
ibm-automation-eventprocessing-v1.3-ibm-operator-catalog-openshift-marketplace   ibm-automation-eventprocessing   ibm-operator-catalog   v1.3

NAME                                                                     PACKAGE                  SOURCE                 CHANNEL
ibm-automation-elastic-v1.3-ibm-operator-catalog-openshift-marketplace   ibm-automation-elastic   ibm-operator-catalog   v1.3

NAME                                                                   PACKAGE                SOURCE                 CHANNEL
ibm-automation-flink-v1.3-ibm-operator-catalog-openshift-marketplace   ibm-automation-flink   ibm-operator-catalog   v1.3

NAME                                                                           PACKAGE                       SOURCE                 CHANNEL
ibm-common-service-operator-v3.23-ibm-operator-catalog-openshift-marketplace   ibm-common-service-operator   ibm-operator-catalog   v3.23

NAME                  PACKAGE               SOURCE                 CHANNEL
ibm-management-kong   ibm-management-kong   ibm-operator-catalog   v3.7

NAME                         PACKAGE                      SOURCE                 CHANNEL
ibm-secure-tunnel-operator   ibm-secure-tunnel-operator   ibm-operator-catalog   v3.7

NAME                           PACKAGE                        SOURCE                 CHANNEL
ibm-watson-aiops-ui-operator   ibm-watson-aiops-ui-operator   ibm-operator-catalog   v3.7

NAME             PACKAGE           SOURCE                 CHANNEL
ir-ai-operator   ibm-aiops-ir-ai   ibm-operator-catalog   v3.7

NAME               PACKAGE             SOURCE                 CHANNEL
ir-core-operator   ibm-aiops-ir-core   ibm-operator-catalog   v3.7

NAME                    PACKAGE                  SOURCE                 CHANNEL
ir-lifecycle-operator   ibm-aiops-ir-lifecycle   ibm-operator-catalog   v3.7

NAME    PACKAGE                              SOURCE                 CHANNEL
redis   ibm-cloud-databases-redis-operator   ibm-operator-catalog   v1.4

NAME    PACKAGE              SOURCE                 CHANNEL
vault   ibm-vault-operator   ibm-operator-catalog   v3.7

______________________________________________________________
Subscriptions from ibm-common-services namespace:

NAME                      PACKAGE                   SOURCE                 CHANNEL
cloud-native-postgresql   cloud-native-postgresql   ibm-operator-catalog   stable

NAME                        PACKAGE                     SOURCE                 CHANNEL
ibm-cert-manager-operator   ibm-cert-manager-operator   ibm-operator-catalog   v3.23

NAME                          PACKAGE                       SOURCE                 CHANNEL
ibm-common-service-operator   ibm-common-service-operator   ibm-operator-catalog   v3.23

NAME                    PACKAGE                     SOURCE                 CHANNEL
ibm-commonui-operator   ibm-commonui-operator-app   ibm-operator-catalog   v3.23

NAME                          PACKAGE                       SOURCE                 CHANNEL
ibm-crossplane-operator-app   ibm-crossplane-operator-app   ibm-operator-catalog   v3.23

NAME                                              PACKAGE                                           SOURCE                 CHANNEL
ibm-crossplane-provider-kubernetes-operator-app   ibm-crossplane-provider-kubernetes-operator-app   ibm-operator-catalog   v3.23

NAME                  PACKAGE               SOURCE                 CHANNEL
ibm-events-operator   ibm-events-operator   ibm-operator-catalog   v3

NAME               PACKAGE            SOURCE                 CHANNEL
ibm-iam-operator   ibm-iam-operator   ibm-operator-catalog   v3.23

NAME                         PACKAGE                          SOURCE                 CHANNEL
ibm-ingress-nginx-operator   ibm-ingress-nginx-operator-app   ibm-operator-catalog   v3.23

NAME                     PACKAGE                      SOURCE                 CHANNEL
ibm-licensing-operator   ibm-licensing-operator-app   ibm-operator-catalog   v3.23

NAME                              PACKAGE                               SOURCE                 CHANNEL
ibm-management-ingress-operator   ibm-management-ingress-operator-app   ibm-operator-catalog   v3.23

NAME                   PACKAGE                    SOURCE                 CHANNEL
ibm-mongodb-operator   ibm-mongodb-operator-app   ibm-operator-catalog   v3.23

NAME                           PACKAGE                        SOURCE                 CHANNEL
ibm-namespace-scope-operator   ibm-namespace-scope-operator   ibm-operator-catalog   v3.23

NAME                        PACKAGE                         SOURCE                 CHANNEL
ibm-platform-api-operator   ibm-platform-api-operator-app   ibm-operator-catalog   v3.23

NAME               PACKAGE            SOURCE                 CHANNEL
ibm-zen-operator   ibm-zen-operator   ibm-operator-catalog   v3.23

NAME                                       PACKAGE    SOURCE                 CHANNEL
operand-deployment-lifecycle-manager-app   ibm-odlm   ibm-operator-catalog   v3.23

______________________________________________________________
OperandRequest instances:

NAMESPACE   NAME             PHASE     CREATED AT
cp4waiops   aiopsedge-base   Running   2023-06-13T18:27:47Z

NAMESPACE   NAME           PHASE     CREATED AT
cp4waiops   aiopsedge-cs   Running   2023-06-13T18:27:47Z

NAMESPACE   NAME                PHASE     CREATED AT
cp4waiops   iaf-core-operator   Running   2023-06-13T18:21:22Z

NAMESPACE   NAME                           PHASE     CREATED AT
cp4waiops   iaf-eventprocessing-operator   Running   2023-06-13T18:21:24Z

NAMESPACE   NAME           PHASE     CREATED AT
cp4waiops   iaf-operator   Running   2023-06-13T18:21:21Z

NAMESPACE   NAME         PHASE     CREATED AT
cp4waiops   iaf-system   Running   2023-06-13T18:27:30Z

NAMESPACE   NAME                        PHASE     CREATED AT
cp4waiops   iaf-system-common-service   Running   2023-06-13T18:24:48Z

NAMESPACE   NAME                                  PHASE     CREATED AT
cp4waiops   operandrequest-kafkauser-iaf-system   Running   2023-06-13T18:54:36Z

NAMESPACE   NAME                   PHASE     CREATED AT
cp4waiops   ibm-aiops-ai-manager   Running   2023-06-13T18:24:44Z

NAMESPACE   NAME                         PHASE     CREATED AT
cp4waiops   ibm-aiops-aiops-foundation   Running   2023-06-13T18:24:44Z

NAMESPACE   NAME                   PHASE     CREATED AT
cp4waiops   ibm-aiops-connection   Running   2023-06-13T18:24:44Z

NAMESPACE   NAME              PHASE     CREATED AT
cp4waiops   ibm-iam-service   Running   2023-06-13T18:48:10Z

NAMESPACE             NAME                   PHASE     CREATED AT
ibm-common-services   ibm-commonui-request   Running   2023-06-13T18:25:05Z

NAMESPACE             NAME              PHASE     CREATED AT
ibm-common-services   ibm-iam-request   Running   2023-06-13T18:25:05Z

NAMESPACE             NAME                  PHASE     CREATED AT
ibm-common-services   ibm-mongodb-request   Running   2023-06-13T18:27:13Z

NAMESPACE             NAME                 PHASE     CREATED AT
ibm-common-services   management-ingress   Running   2023-06-13T18:27:13Z

NAMESPACE             NAME                   PHASE     CREATED AT
ibm-common-services   platform-api-request   Running   2023-06-13T18:27:15Z

______________________________________________________________
ODLM pod current status:

ibm-common-services                                operand-deployment-lifecycle-manager-54747bcb5-w9hb9              1/1     Running     0               131m
______________________________________________________________
Orchestrator pod current status:

cp4waiops                                          ibm-aiops-orchestrator-controller-manager-64bcb6b9fb-npzmk        1/1     Running     0               131m
```
